docs/02-membership.md

```diff
@@ -276,6 +249,78 @@ The table and graphs below illustrate how the 5 year weight change of a hypothet
 
 ![RelativeWeights](https://github.com/cheeky-gorilla/membership/assets/76262359/93a6f5a0-393e-46c8-8dfd-a51aa6864f46)
 
+### Governance
+
+Only members are invited to govern the mechanism. This may include:
+
+- modifying high-level project elibility
+- discussing which smart contracts to use, and how to operate them
+- setting parameters like how long vests should be
+- how and where to fundraise
+
+Typically this takes the shape of rough consensus discussions. In some special cases, decisions needs to be voted on or ratified - learn more in smart contract section.
+
+## 2.4 Obligations
+
+Members are expected to participate in curation. Consider these examples:
+
+- an individual updating their personal status
+- a colleague making updates on behalf of their team
+- members giving input to shape the addition/removal process
+
+### Self-Curation
+
+We use the term "self-curation" to describe how the membership curates its own beneficiary set. This is an important part of the funding mechanism, and what distinguishes it from others e.g. badgeholders in OP Retro, or Gitcoin's use of donors and quadratic weighting. Curatorial bodies external to the beneficiary set come with their own tradeoffs. Self-curation is suited for this context for a few reasons:
```

Comment:
Avoid the circular definition, and consider removing explicit references to other mechanisms. 

```suggestion
We use the term "self-curation" to describe how the membership selects its own beneficiary set. This is an important distinction between Protocol Guild and other public goods funding mechanisms. While curatorial bodies external to the beneficiary set are appropriate in some contexts, self-curation is well suited here for a few reasons:
```